### ISSUE TITLE
Look into using JsSerializeScript to boost load times

### ISSUE THREAD

**Issue by rozele** [ReactWindows/react-native-windows#585]
This pre-parses the JavaScript and can save precious milliseconds during load time.

**Comment by rozele** [comment]
Fixes #291

**Comment by rozele** [comment]
I ran a test that loaded the UIExplorerApp using the raw JavaScript text and the serialized version of the JavaScript text (from the JsSerializeScript API).  I ran the test 50 times for each.

For the raw JavaScript text the load times were:
Min: 94.2 ms
Max: 195.2 ms
Avg: 114.8 ms 

For the pre-serialized JavaScript:
Min: 20.2 ms
Max: 41.7 ms
Avg: 27.4 ms

This is a savings of an average of 87 ms from the load time of the application.  There is one blocking issue, namely that we do not know how large of a buffer to instantiate (there is no guaranteed correlation between JavaScript text size and serialized script size).  I have a thread out with the Chakra team to get feedback on this approach.
